### PR TITLE
Improve pooling

### PR DESCRIPTION
**Pull request by AndrewAR2**
# Checklist:

- [x] Code follows the style guidelines
- [ ] Documentation updated
- [ ] Corner cases evaluated and handled
- [ ] index.js exports updated
- [ ] Automatic tests added/updated
- [x] Automatic tests pass
- [x] Sanity check passed
- [x] Self-review passed
- [ ] Version updated

### CODE REVIEW

File: contracts/Pool/Pool.sol [hunk 1]
```diff
@@ -31,6 +32,7 @@ contract Pool is Ownable {
     string public name;
     uint public id;
 
+    // todo: Is it really necessary?
```
Comment (vitalii427) [inline contracts/Pool/Pool.sol:35]: It is only needed for _getManager().retire(). We can add some named payable function and change manager so it will use it.

**Review by vitalii427 — APPROVED** [review]
